packages/dashboard/README.md

```diff
@@ -134,18 +170,21 @@ Once resolved, we will be able to see basic information about the DID broken dow
 3. Set the variables in .env according to how you want your hosted sidetree setup.
 4. `docker-compose build`
 5. `docker-compose up`
-6.  At this point you should be able to test sidetree locally, to go to http://localhost:8080 in your browser.
+6. At this point you should be able to test sidetree locally, to go to http://localhost:8080 in your browser.
```

Comment:
Port 8080 or 3000?

Reply:
8080 for docker , gcp cloud run uses this port as default